index.html

```diff
@@ -25,7 +25,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, shrink-to-fit=no">
         <meta name="apple-mobile-web-app-capable" content="yes">
-        <title></title>
+        <title>test</title>
```

Comment:
Is this left here by mistake?

Reply:
Yep. Good catch, thanks. Fixed and committed.